EMML/2001-3000/EMML2584.xml

```diff
@@ -32,11 +32,102 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
             <msDesc xml:id="ms">
                <msIdentifier>
                   <repository ref="INS0004HMML"/>
-                  <collection>Parish Church, Ankober</collection>
```

Comment:
Thank you, Marcin. I think that it has been decided to substitute the HMML rep with the real repository of the manuscripts and keep information on the  HMML in the collection. Here in the example 2: https://betamasaheft.eu/Guidelines/?q=EMML&id=identifiersStructure

Reply:
Done!